engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/details/subscription-details.controller.js

```diff
@@ -51,5 +51,7 @@ angular.module('Bastion.subscriptions').controller('SubscriptionDetailsControlle
 
             return "";
         };
+
+        $scope.virtWhoToolTip = translate("If the virt-who field is Yes then the subscription requires the use of virt-who. Learn how to configure and use this tool in the <a href=\"https://access.redhat.com/documentation/en/red-hat-satellite/6.2/single/virtual-instances-guide/\">Virtual Instances Guide</a>.");
```

Comment:
I have not been able to get this PR to work, but this would be my suggestion: Change the column title to "virt-who Functionality". In that column, for the primary subscriptions put text like "Yes". For guest subscriptions put "virt-who required".